docs/data/base/getting-started/usage/usage.md

```diff
@@ -20,60 +20,68 @@ function App() {
 ```
 
 You can play around with this code in the interactive Code Sandbox demo below.
-Try importing an `InputUnstyled` component and adding it to the `<div>`:
+Try importing an [`InputUnstyled`](/base/react-input/) component and adding it to the `<div>`:
 
 {{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
 ## Shared props
```

Comment:
Is it better if this changes to `Slots`?

Reply:
I would expect "slots" to refer to the actual slots themselves—to me, this section is mainly about the props that are used to customize the slots, that are shared by all components.